### ISSUE TITLE
KDE - System tray Icon is resized incorrectly on KDE

### ISSUE THREAD

**Issue by Nohus** [dorkbox/SystemTray#67]
The tray icon looks deformed on KDE Plasma 5.8.6:

![bug](https://user-images.githubusercontent.com/11064768/32244570-dccce83c-be70-11e7-81ff-165ca72d4381.png)

The icon image has perfectly straight edges, but it looks deformed like in the screenshot.

I tried 32x32 and 256x256 icons, thinking it was a resizing issue, but it happens at both sizes.

**Comment by dorkbox** [comment]
can you include an output of the log?

**Comment by Nohus** [comment]
Here it is:
```
23:05:40.751 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Forcing GTK2 because JavaFX is GTK2
23:05:40.754 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - OS: Linux
23:05:40.757 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Arch: amd64
23:05:40.757 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Oracle Corporation Java HotSpot(TM) 64-Bit Server VM 1.8.0_144
23:05:40.757 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Is Auto sizing tray/menu? true
23:05:40.757 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Is JavaFX detected? true
23:05:40.757 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Is SWT detected? false
23:05:40.817 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Java Swing L&F: Metal
23:05:40.817 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Auto-detecting tray type
23:05:40.817 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Force GTK2: true
23:05:40.817 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Prefer GTK3: true
23:05:40.956 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Currently using the 'KDE' desktop environment
23:05:41.309 [JavaFX Application Thread] DEBUG dorkbox.util.jna.linux.GtkLoader - GTK: gtk-x11-2.0
23:05:41.312 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - GTK Version: 2.24.31
23:05:41.313 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Is the system already running GTK? true
23:05:41.583 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Tray indicator image size: 22
23:05:41.583 [JavaFX Application Thread] DEBUG dorkbox.systemTray.SystemTray - Tray menu image size: 16
23:05:41.653 [JavaFX Application Thread] INFO dorkbox.systemTray.SystemTray - Successfully loaded type: _AppIndicatorNativeTray
```

**Comment by dorkbox** [comment]
This should be fixed in Version 3.17, which has just released. Also - there is more detailed log information when tray image sizing cannot be auto-detected (which is likely the culprit here)
